app/src/androidTest/java/com/github/sdp/tarjetakuna/utils/PermissionGranting.kt

```diff
@@ -0,0 +1,30 @@
+package com.github.sdp.tarjetakuna.utils
+
+import android.os.Build
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.uiautomator.UiDevice
+import androidx.test.uiautomator.UiSelector
+
+class PermissionGranting {
+    object PermissionGranting {
+        fun grantPermission() {

```

Comment:
Is this taken from somewhere? It seems like the kind of wheel you should not need to reinvent

Reply:
yes it s coming from here : https://alexzh.com/ui-testing-of-android-runtime-permissions/ 
i ll add the source on the code.
They also show another way to grant permission but it keeps failing saying that the permission cannot be granted :
@get:Rule
val permissionRule = GrantPermissionRule.grant( android.Manifest.permission.WRITE_EXTERNAL_STORAGE)
This fails while running the test in JUnit